### PR TITLE
Ingest humeval 2021 (part of EACL 2021)

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

**Comment by akoehn** [comment]
Will merge after checks are complete. This live preview is so helpful!

**Comment by mjpost** [comment]
Yes, it's really nice! Thanks for merging.

**Comment by akoehn** [comment]
hmm, the humeval is still not linked from EACL 2021.

**Comment by mjpost** [comment]
It is for me—I think you just have a caching issue.

**Review by akoehn — APPROVED** [review]
You are right; everything is good

File: bin/ingest.py [hunk 1]
```diff
@@ -245,8 +245,8 @@ def main(args):
             os.makedirs(pdfs_dest_dir)
 
         # copy the book
-        book_src_filename = meta["abbrev"] + "-" + year
-        book_src_path = os.path.join(root_path, book_src_filename) + ".pdf"
+        book_src_filename = f'{year}.{meta["abbrev"]}-{volume_name}.pdf'
+        book_src_path = os.path.join(root_path, book_src_filename)
         book_dest_path = None
```
Comment (mjpost) [inline bin/ingest.py:250]: We were using the wrong pattern to find the books, with the result that many recent books were not committed. This fixes that.

**Pull request merged**